libs/design/scss/daff-typography.scss

```diff
@@ -11,20 +17,18 @@
 //
 // @usage
 // ```
-// @import "daff-typography";
+// @use '@daffodil/design/scss/daff-typography';
 // ```
 
-@import './responsive/variables';
-@import './responsive/breakpoint';
-@import './typography/variables';
-@import './typography/mixins';
-@import './typography/sizes/mixins';
+@use '~modern-normalize/modern-normalize';
 
-@import '~modern-normalize/modern-normalize.css';
+@use 'typography' as t;
+@forward 'typography';
+@import 'typography';
```

Comment:
This still looks like an `@import`?

Reply:
This `@import` is to prevent a breaking change. It'll be removed in v1.0.0

Reply:
removed